File: codes/c/calculator/calculator.c
#include<stdio.h>

void main()
{
    int first, second, answer, remainder;
    char operator;
    printf("\nEnter operator (+, -, *, /): ");
    scanf("%c", &operator);
    if(operator != '+' && operator != '-' && operator != '*' && operator != '/')
    {
        printf("Invalid operator.");
    }
    else
    {
        printf("Enter the first number: ");
        scanf("%d", &first);
        printf("Enter the second number: ");
        scanf("%d", &second);
        switch(operator)
        {
                case'+':
                        answer = first + second;
                        printf("Sum: %d", answer);
                        break;
                case'-':
                        answer = first - second;
                        printf("Difference: %d", answer);
                        break;
                case'*':
                        answer = first * second;
                        printf("Product: %d", answer);
                        break;
                case'/':
                        answer = first / second;
                        remainder = first % second;
                        printf("Quotient: %d", answer);
                        printf("\nRemainder: %d", remainder);
                        break;
        }
    }
    printf("\n\n");
}

File: codes/python/calculator/calculator.py
operator = input("Enter operator (+, -, *, /): ")

if operator == str("+") or operator == ("-") or operator == ("*") or operator == ("/"):
    first = input("First number: ")
    second = input("Second number: ")

    if operator == str("+"):
        print("Sum: " + str(int(first) + int(second)))

    elif operator == str("-"):
        print("Difference: " + str(int(first) - int(second)))

    elif operator == str("*"):
        print("Product: " + str(int(first) * int(second)))

    elif operator == str("/"):
        print("Quotient: " + str(int(int(first) / int(second))))
        print("Remainder: " + str(int(first) % int(second)))

else:
    print("Invalid operator.")